packages/amplify_core/lib/src/state_machine/state_machine.dart

```diff
@@ -286,7 +294,7 @@ abstract class StateMachine<
     final precondError = event.checkPrecondition(currentState);
     if (precondError != null) {
       logger.debug(
-        'Precondition not met for event ($event):\n'
+        'Precondition not met for event ($_currentEvent):\n'
```

Comment:
why changing to log `_currentEvent` instead of `event`?

Reply:
Whoops - leftover from experiment. Fixed.